assets/scripts/gallery/view.js

```diff
@@ -196,7 +196,7 @@ export function receiveGalleryData (transmission) {
 
     // This adds the street name plaque to each thumbnail.
     // the variable is assigned, but not re-used. Do not remove!
-    var streetName = new StreetName(nameEl, galleryStreet.name)
+    var streetName = new StreetName(nameEl, galleryStreet.name) // eslint-disable-line no-unused-vars
```

Comment:
Standard uses `eslint` under the hood, so yes, the same notation that `eslint` uses will override the settings set by `standard`.